### DISCUSSION TITLE
[Bug]: pimcore 11 Insallation on windows gives error

### DISCUSSION THREAD

**Discussion by varshaekrocx** [pimcore/pimcore#16708] [General]
### Pimcore version

11

### Steps to reproduce

Download pimcore 11 on xampp 8.2 using the below link
https://github.com/pimcore/skeleton/?tab=readme-ov-file

### Actual Behavior

All installation process is done but while running it locally it gives error
![image](https://github.com/pimcore/pimcore/assets/124252930/0dfcaf87-346e-4ed8-9e3a-14c1b5e2f61c)

### Expected Behavior

After complete installation it should show the pimcore admin interface

**Comment by blankse** [comment]
The Locale class is part of the intl php module. Seems you haven't install it.
https://www.php.net/manual/en/class.locale.php

Make sure you have all required php module installed:
https://pimcore.com/docs/platform/Pimcore/Installation_and_Upgrade/System_Requirements/#required-settings-and-modules--extensions

**Comment by varshaekrocx** [comment]
> The Locale class is part of the intl php module. Seems you haven't install it. https://www.php.net/manual/en/class.locale.php
> 
> Make sure you have all required php module installed: https://pimcore.com/docs/platform/Pimcore/Installation_and_Upgrade/System_Requirements/#required-settings-and-modules--extensions

Thanks @blankse it worked.
